.gitattributes

```diff
@@ -12,6 +12,13 @@
 /phpunit.xml.dist export-ignore
 /src/Fixer/Internal/ export-ignore
 /tests/ export-ignore
+/feature-or-bug.rst export-ignore
+/logo.md export-ignore
+/logo.png export-ignore
+/CHANGELOG.md export-ignore
+/ci-integration.sh export-ignore
```

Comment:
What is the purpose of `ci-integration.sh`? Is it maybe intended for use in projects? (not sure)

Reply:
#8989